lightningd/channel/channel.c

```diff
@@ -2002,6 +2032,7 @@ static void init_channel(struct peer *peer)
 				   &peer->short_channel_ids[LOCAL],
 				   &reconnected,
 				   &peer->unsent_shutdown_scriptpubkey,
+				   &peer->shutdown_sent[REMOTE],
```

Comment:
It's passed as "peer->remote_shutdown_scriptpubkey != NULL" from peer_control.c.